src/test/acceptance/artist.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env mocha */
+import { setup, teardown } from "./helpers"
+
+describe("Artist page", () => {
+  let metaphysics, browser
+
+  before(async () => {
+    ;({ metaphysics, browser } = await setup())
+    metaphysics.post("/", (req, res) => {
+      res.send(require("./fixtures/metaphysics/artist"))
+    })
```

Comment:
Oh interesting -- we're using fixture data throughout all the acceptence tests. Was this done for speed / stability reasons?  Why don't we access the real page via url and work from there?

Reply:
Yea this will actually spin up servers for gravity, positron, metaphysics and you stub the response.

By real page, do you mean a test which accesses staging and asserts things? We've tried that twice before, latest in https://github.com/artsy/vapor . It's a very valid and worthy idea, to at least ensure all top level pages are functional. We found that it was brittle overall (for instance we probably wouldn't want to block CI on that), it wasn't kept up to date as Force changed, and it just led to lots of false positives.

Reply:
> it wasn't kept up to date as Force changed

I would think that the tests themselves should be updated in that case, like we don't need to test every little thing but get in some key existence checks. Errors there would seem like something to be concerned about. 

> We found that it was brittle overall (for instance we probably wouldn't want to block CI on that)

This relates to my q about Puppetter vs PhantomJS / more recent tech and if it might make sense to try to upgrade Nightmare and give it a try.

Reply:
> I would think that the tests themselves should be updated in that case, like we don't need to test every little thing but get in some key existence checks. Errors there would seem like something to be concerned about.

You'd be surprised at just how fast and easy this winds up getting dropped and ignored, in the sea of notifications and alert noise we live in.

> This relates to my q about Puppetter vs PhantomJS / more recent tech and if it might make sense to try to upgrade Nightmare and give it a try.

Maybe vapor is worth resurrecting with newer tech and seeing what to do? I would merge this PR in the meantime since it does add a layer of integration testing we did not have. The ideal to also be able to _actually_ smoke test against staging or production (either here, within vapor, or elsewhere) would be great to work on and not lose track of.

Reply:
Wouldn’t this test be brittle in the sense that anytime some Reaction component changes its data requirement this test over in Force could fail simply due to the stubbed data being incomplete to satisfy that change?

Reply:
If the Reaction component wasn't defensive as to the missing data (depending on what it is), then yes, definitely. When Reaction gets bumped in Force you might get that fail.

That is indeed possibly going to be brittle! I guess, we can just see how we feel about it if/when it happens? We can always remove the test if it's not adding much value.

Seems like integration tests might be somewhat brittle by their nature, plus might be able to smoke test not just Force, but possibly CMS and other apps. So if/when we want to resurrect vapor, we should probably do that in a general way in its own repo (not Force).

Reply:
Absolutely fair. I guess I was mostly thinking of people updating Reaction in Force and being faced by this being caused by changes not made by them, while the version bump not being a major  one. But true, we’ll see how that goes 👍